ui/apps/platform/cypress/helpers/networkGraph.js

```diff
@@ -30,9 +30,7 @@ export function clickOnNodeByName(cytoscape, node) {
 }
 
 export function clickOnDeploymentNodeByName(cytoscape, name) {
-    cy.intercept('GET', api.network.deployment).as('getDeployment');
     clickOnNodeByName(cytoscape, { type: 'DEPLOYMENT', name });
-    cy.wait('@getDeployment');
 }
```

Comment:
Aha, as a follow up for me after this is merged to undo part of what I did in #1239
* replace `clickOnDeploymentNodeByName` calls
* with `clickOnNameByName`

Reply:
Ah wow I didn't realize this code was something you just added.

I am hopeful that the code removal here doesn't end up causing e2e test failures. Even though the network requests were unneeded, they did impact timing in Cypress tests. Everything was passing for me locally, so hopefully that continues to be the case in CI.

Reply:
Running this branch, there is a /v1/deployments/:id request for the **first** click on a deployment node.

So the following intercept, click, wait sandwich works, although I will verify as follow up whether it is needed:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/cypress/integration/networkGraph/networkGraph.test.js#L20-L27

Reply:
Not unique to this pull request, I see some errors when using browser history to go back to main dashboard.
![back](https://user-images.githubusercontent.com/11862657/163046785-153d320a-7f35-404e-8d8f-8382d925abef.png)